### ISSUE TITLE
Upgrade for card with reduced Immersion cost loses reduction

### ISSUE THREAD

**Issue by simply-peachy** [db0/hypnagonia#204]
I chose a Curio which causes a card to cost zero Immersion. Once the card accrued enough points to upgrade, both of the upgrade choices meant the card would cost 1 Immersion.

Original card: Interpretation: 1 Immersion cost for 6 Interpretation.
After Curio: 0 Immersion cost for 6 Interpretation.
Upgrade choices offered: 1 Immersion cost for 9 Interpretation or 1 Immersion cost for 12 Interpretation, Release.

**Comment by db0** [comment]
Did you check that the card had 1 immersion after being upgraded? Normally the upgraded card should cost 0, even if the upgrade choices show 1.

**Comment by simply-peachy** [comment]
No, I just saw the upgrade options and chose to leave the card as-is. I'll try choosing an upgrade next time.
